### PR TITLE
Staging 0.35.0

### LINKED ISSUE #1909: [2] Upgrade trivy-operator
**What Chart should be upgraded?**:

upstream / trivy-operator https://github.com/aquasecurity/trivy-operator/releases/tag/v0.17.1

**Why should this Chart be upgraded?**:

- [ ] We need to perform a security upgrade.
- [ ] We need it for a new feature: *which feature (link to blocked issue)*.
- [x] We need to keep up to prevent future issues.
- [ ] Other: *Write the reason here*

**Acceptance criteria**:

- I checked the migration of the new Chart:
  - [ ] I upgraded a Chart and determined that no migration steps are needed.
  - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).
- [ ] I tested the functionality provided by the new Chart (e.g., Auth flow, Grafana dashboards, etc.)

### PR DESCRIPTION
**Pull request by crssnd**
> [!warning]
> **This is public repository, ensure not to disclose:**
> - [x] personal data beyond what is necessary for interacting with this pull request
> - [x] business confidential information, such as customer names

### What kind of PR is this?

**Required**: Mark one of the following that is applicable:

- [ ] kind/feature       
- [x] kind/improvement   
- [ ] kind/deprecation   
- [ ] kind/documentation 
- [ ] kind/clean-up      
- [ ] kind/bug           
- [ ] kind/other         

*Optional*: Mark one or more of the following that are applicable:

> [!important]
> Breaking changes should be marked `kind/admin-change` or `kind/dev-change` depending on type
> Critical security fixes should be marked with `kind/security`

- [ ] kind/admin-change  
- [ ] kind/dev-change    
- [ ] kind/security      
- [ ] [kind/adr]()       

### What does this PR do / why do we need this PR?

- Fixes #1909

#### Checklist

- [ ] Proper commit message prefix on all commits
  
- Change checks:
  - [ ] The change is transparent
  - [ ] The change is disruptive
  - [ ] The change requires no migration steps
  - [x] The change requires migration steps
- Metrics checks:
  - [ ] The metrics are still exposed and present in Grafana after the change
  - [ ] The metrics names didn't change (Grafana dashboards and Prometheus alerts are not affected)
  - [ ] The metrics names did change (Grafana dashboards and Prometheus alerts were fixed)
- Logs checks:
  - [ ] The logs do not show any errors after the change
- Network Policy checks:
  - [ ] Any changed pod is covered by Network Policies
  - [ ] The change does not cause any dropped packages in the `NetworkPolicy Dashboard`
- Pod Security Policy checks:
  - [ ] Any changed pod is covered by Pod Security Admission
  - [ ] Any changed pod is covered by Gatekeeper Pod Security Policies
  - [ ] The change does not cause any pods to be blocked by Pod Security Admission or Policies
- Falco checks:
  - [ ] The change does not cause any alerts to be generated by Falco
- Audit checks:
  - [ ] The change does not cause any unnecessary Kubernetes audit events
  - [ ] The change requires changes to Kubernetes audit policy
- Bug checks:
  - [ ] The bug fix is covered by regression tests

### CODE REVIEW

**Comment by simonklb** [comment]
The `bug:` prefix that you removed in https://github.com/elastisys/compliantkubernetes-apps/pull/1934/commits/642554b1bd973d6d8d65adcc288a6ccb4301a105 is actually added by the changelog generator and was not part of the PR title.

It's added to distinguish different kinds of issues under the `Other(s)` category. See the initial changelog implementation task: https://github.com/elastisys/ck8s-issue-tracker/issues/79

If you disagree with it feel free to raise an issue!

File: helmfile/charts/grafana-dashboards/files/welcome.md [hunk 1]
```diff
@@ -4,11 +4,13 @@
 
 Here you can find the most relevant features and changes for the last couple of releases of Compliant Kubernetes
 
+- Harbor was upgraded to v2.9.1. As part of this upgrade, Notary V1 is removed. **[v0.35]**
+- Chroot for ingress-nginx is enabled by default. **[v0.35]**
```
Comment (viktor-f) [inline helmfile/charts/grafana-dashboards/files/welcome.md:8]: I wonder if the line about chroot should be reworded. I think it could be hard for app devs to understand what it means if nginx is running with chroot. Could you reword it to explain what effects it has for the app devs instead?
Same comment for the welcome dashboard in opensearch.

Comment (crssnd) [reply helmfile/charts/grafana-dashboards/files/welcome.md]: pushed some changes

**Review by viktor-f — APPROVED** [review]

**Review by raviranjanelastisys — APPROVED** [review]
LGTM !!

**Review by lunkan93 — APPROVED** [review]

**Review by aarnq — APPROVED** [review]

**Review by viktor-f — APPROVED** [review]

**Review by lunkan93 — APPROVED** [review]
